src/dynarec/rv64/dynarec_rv64_660f_vector.c

```diff
@@ -655,12 +655,62 @@ uintptr_t dynarec64_660F_vector(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t i
             VRGATHER_VV(d1, v0, q1, VECTOR_UNMASKED);
             VMERGE_VVM(q0, d1, d0);
             break;
-        case 0x66:
-            INST_NAME("PCMPGTD Gx, Ex");
+        case 0x62:
+            INST_NAME("PUNPCKLDQ Gx,Ex");
             nextop = F8;
             SET_ELEMENT_WIDTH(x1, VECTOR_SEW32, 1);
+            ADDI(x1, xZR, 0b1010);
+            VMV_V_X(VMASK, x1); // VMASK = 0b1010
+            v0 = fpu_get_scratch(dyn);
+            VIOTA_M(v0, VMASK, VECTOR_UNMASKED); // v0 = 1 1 0 0
             GETGX_vector(q0, 1, VECTOR_SEW32);
             GETEX_vector(q1, 0, 0, VECTOR_SEW32);
+            d0 = fpu_get_scratch(dyn);
+            d1 = fpu_get_scratch(dyn);
+            VRGATHER_VV(d0, v0, q0, VECTOR_UNMASKED);
+            VRGATHER_VV(d1, v0, q1, VECTOR_UNMASKED);
+            VMERGE_VVM(q0, d1, d0);
+            break;
+        case 0x63:
+            INST_NAME("PACKSSWB Gx, Ex");
+            nextop = F8;
+            SET_ELEMENT_WIDTH(x1, VECTOR_SEW16, 1);
+            GETGX_vector(q0, 1, VECTOR_SEW16);
+            GETEX_vector(q1, 0, 0, VECTOR_SEW16);
+            fpu_get_scratch(dyn); // HACK: skip v3, for vector register group alignment!
```

Comment:
I don't understand this. Especialy because EX can be a memory and so a scratch register already.

Reply:
if you need d0/d1 to be consecutive and with some other constraint, create a function to grab 2 consecutive scratch instead (I think there is one on arm64 already)

Reply:
Indeed, it will be wrong if EX is from memory, I'll fix this as you suggested, merci!

Reply:
Done.